_posts/2020-06-30-Making-IrishGen-SPARQL-Part-II.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+date: 2020-06-30
+title: "Making IrishGen SPARQL Part II: Constructs"
+author: cgy
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+The first post in this series covered the `select` query form and used
+individuals with the name Báeth as a guide.  We will continue our
+sojourn through SPARQL with Báeth and his similarly named compatriots
```

Comment:
You need to make it clear that there is not just one individual called Báeth but that you are initially talking about all individuals called Báeth.